docs/manual/manual_timeout.rst

```diff
@@ -0,0 +1,158 @@
+	| Vertiq 4006 | .. centered:: |:white_check_mark:| |
+	+-------------+------------------------------------+
+	| Vertiq 2306 | .. centered:: |:white_check_mark:| |
+	+-------------+------------------------------------+
+
+Servo Modules
+**************
+Servo modules do not support the full range of configurable timeout features described in this section. They do support a timeout, and the period of the timeout is configurable.
+The timeout behavior, timeout song playback options, and timeout meaning described in this section are not supported on servo modules. They will always 
+continuously play the timeout song when they timeout.
+
+.. table:: Servo Module Support for Configurable Timeout
+
+	+-------------+------------------------------------+-------------------------------------------------------------------------+
+	| Module      | Configurable Timeout Support       | Notes                                                                   |
+	+-------------+------------------------------------+-------------------------------------------------------------------------+
+	| Vertiq 8108 | .. centered:: |:warning:|          | A limited subset of timeout features are supported, see the note above. |
+	+-------------+------------------------------------+-------------------------------------------------------------------------+
+	| Vertiq 4006 | .. centered:: |:warning:|          | A limited subset of timeout features are supported, see the note above. |
+	+-------------+------------------------------------+-------------------------------------------------------------------------+
+	| Vertiq 2306 | .. centered:: |:warning:|          | A limited subset of timeout features are supported, see the note above. |
+	+-------------+------------------------------------+-------------------------------------------------------------------------+
+
+Configuring Timeout
+====================
+
+.. _timeout_period:
+
+Timeout Period
+***************
+The length of time between messages that will cause a timeout to occur can be configured by the user. This can be set using the *Timeout* parameter under the 
+Tuning tab of the IQ Control Center as shown below. If the module is spinning and a new message is not received in the time set by this configuration parameter, 
+a timeout will trigger. See the :ref:`triggering_timeout` section for more information.
+
+.. figure:: ../_static/manual_images/timeout/timeout_parameter.png
+    :align: center
+    :width: 60%
+    :alt: Timeout Parameter
+
+    Timeout Parameter in IQ Control Center
+
+.. _timeout_behavior:
+
+Timeout Behavior
+*****************
+On modules that support it (see :ref:`timeout_module_support` above), the behavior of the module when a timeout occurs can be configured. If a module does not support 
+configurable timeout behavior, it will always coast and then play its timeout song continuously when a timeout occurs. **Also, the module will always disarm 

```

Comment:
again, need to check on if this is actually true...I know that they coast, but I don't know if they actually play anything.